src/Servers/Kestrel/Core/test/TlsListenerMiddlewareTests.cs

```diff
@@ -66,34 +66,21 @@ public async Task RunTlsClientHelloCallbackTest_DeterministicallyReads()
         var transport = new DuplexPipe(reader, writer);
```

Comment:
Add cancellation tests? One where the token is pre-canceled, and one where we cancel during a pending `ReadAsync` call.

Reply:
added 2 tests